.env.example

```diff
@@ -41,8 +41,12 @@ CKAN_SMTP_MAIL_FROM=ckan@localhost
 
 # Extensions
 
-CKAN__PLUGINS=envvars image_view text_view recline_view datastore datapusher
+CKAN__PLUGINS=envvars image_view text_view recline_view datastore saml2
```

Comment:
I have an error with saml2. It's not installed

Reply:
I would like to confirm whether we actually want the `datastore` plugin.

Reply:
@woodt I'll remove `saml2` and `datastore` for now, we can add whatever we need as we need